airflow/providers/hashicorp/secrets/vault.py

```diff
@@ -31,7 +31,7 @@
 
 
 class VaultSecrets(BaseSecretsBackend, LoggingMixin):
-    """
+    """ # noqa pylint: disable=line-too-long
```

Comment:
Let's try to avoid `disable=line-too-long` :)

Reply:
Fixed as suggested